homeassistant/components/geonetnz_quakes/__init__.py

```diff
@@ -96,6 +122,113 @@
     manager = hass.data[DOMAIN][FEED].pop(config_entry.entry_id)
     await manager.async_stop()
 
-    await hass.config_entries.async_forward_entry_unload(config_entry, "geo_location")
+    for domain in COMPONENTS:
+        hass.async_create_task(
```

Comment:
We should await all the tasks using either `asyncio.wait` or `asyncio.gather`.

Reply:
OK, changed in this case.
A similar change I tried for setting up platforms does not work and leads to the setup freezing.

Reply:
Yeah, we can only await when unloading config entry, not when forwarding config entry.